elements/howto-tabs/howto-tabs.js

```diff
@@ -47,54 +52,55 @@
 
       this.setAttribute('role', 'tablist');
 
-      // Before the elements starts booting, it waits for
-      // the both `<howto-tab>` and `<howto-tabpanel>` to load.
+      // For progressive enhancement, the markup should alternate between tabs
+      // and panels. Elements reordering children tend to not work well with
+      // frameworks. Instead ShadowDOM is used to reorder the elements by
+      // using slots.
+      this.shadowRoot.innerHTML = `
+        <slot name="tab"></slot>
```

Comment:
actually you probably want the `_mounted` stuff so the element doesn't stamp its template over and over.